pkg/tree/breadth_first_walker.go

```diff
@@ -1,79 +0,0 @@
-package tree
```

Comment:
this is here as a utility function for callers to work with the tree readers, though it is not used here it's made available for completeness of the API